packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-foot-note.tsx

```diff
@@ -13,7 +13,15 @@ const FooterNote = ({
     should_restrict_bvi_account_creation,
     should_restrict_vanuatu_account_creation,
 }: TJurisdictionModalFootNoteProps) => {
-    const account_type_name = account_type === 'synthetic' ? 'Derived' : 'Financial';
+    let account_type_name;
+
+    if (account_type === 'synthetic') {
```

Comment:
This can be reused in so many places in CFD package, let's move it to a config file and reuse it.
You can take the titles from cfd.ts CFD_text.

Reply:
Done